manifests/meshnet/grpc/manifest.yaml

```diff
@@ -0,0 +1,195 @@
+                    peer_intf:
+                      description: Peer interface name
+                      type: string
+                    peer_ip:
+                      description: (Optional) Local IP address
+                      type: string
+                    peer_pod:
+                      description: Name of the peer pod
+                      type: string
+                    uid:
+                      description: Unique identified of a p2p link
+                      type: integer
+                  required:
+                  - uid
+                  - peer_pod
+                  - local_intf
+                  - peer_intf
+                  type: object
+                type: array
+            type: object
+          status:
+            properties:
+              net_ns:
+                description: Network namespace of the POD
+                type: string
+              skipped:
+                description: List of pods that are skipped by local pod
+                items:
+                  description: peer pod name
+                  type: string
+                type: array
+              src_ip:
+                description: Source IP of the POD
+                type: string
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: meshnet
+  name: meshnet
+  namespace: meshnet
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: meshnet
+  name: meshnet-clusterrole
+rules:
+- apiGroups:
+  - networkop.co.uk
+  resources:
+  - topologies
+  verbs:
+  - '*'
+- apiGroups:
+  - networkop.co.uk
+  resources:
+  - topologies/status
+  verbs:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: meshnet
+  name: meshnet-clusterrolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: meshnet-clusterrole
+subjects:
+- kind: ServiceAccount
+  name: meshnet
+  namespace: meshnet
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: meshnet
+    k8s-app: meshnet
+  name: meshnet
+  namespace: meshnet
+spec:
+  selector:
+    matchLabels:
+      app: meshnet
+      name: meshnet
+  template:
+    metadata:
+      labels:
+        app: meshnet
+        name: meshnet
+    spec:
+      containers:
+      - env:
+        - name: HOST_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+        - name: INTER_NODE_LINK_TYPE
+          value: GRPC
+        image: us-west1-docker.pkg.dev/kne-external/kne/networkop/meshnet:v0.3.1
+        imagePullPolicy: IfNotPresent
+        name: meshnet
+        resources:
+          limits:
+            memory: 1000Mi
+          requests:
+            cpu: 100m
```

Comment:
Just curious... what does 100m mean for CPU?

Reply:
100 milliCPU

https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit